src/rust/engine/logging/src/logger.rs

```diff
@@ -26,24 +26,24 @@ const TIME_FORMAT_STR: &str = "%H:%M:%S";
 pub type StdioHandler = Box<dyn Fn(&str) -> Result<(), ()> + Send>;
 
 lazy_static! {
-  pub static ref LOGGER: Logger = Logger::new();
+  pub static ref PANTS_LOGGER: PantsLogger = PantsLogger::new();
 }
 
-pub struct Logger {
+pub struct PantsLogger {
   pantsd_log: Mutex<MaybeWriteLogger<File>>,
   stderr_log: Mutex<MaybeWriteLogger<Stderr>>,
   use_color: AtomicBool,
   show_rust_3rdparty_logs: AtomicBool,
   stderr_handlers: Mutex<HashMap<Uuid, StdioHandler>>,
 }
 
-impl Logger {
-  pub fn new() -> Logger {
-    Logger {
+impl PantsLogger {
+  pub fn new() -> PantsLogger {
+    PantsLogger {
       pantsd_log: Mutex::new(MaybeWriteLogger::empty()),
       stderr_log: Mutex::new(MaybeWriteLogger::empty()),
       show_rust_3rdparty_logs: AtomicBool::new(true),
-      use_color: AtomicBool::new(true),
+      use_color: AtomicBool::new(false),
```

Comment:
I figure it's safer to default to not using color until we know for certain that it's safe to enable.